dist/index.d.ts

```diff
@@ -3175,7 +3201,27 @@ interface GPUSupportedLimits {
   readonly maxSampledTexturesPerShaderStage: number;
   readonly maxSamplersPerShaderStage: number;
   readonly maxStorageBuffersPerShaderStage: number;
+  /**
+   * @todo Temporarily optional until all browsers have implemented it.
+   * Check support before using. (Use the non-null assertion operator `!` where needed.)
+   */
+  readonly maxStorageBuffersInVertexStage?: number;
```

Comment:
I thought we determined in the last meeting that if it's in the spec implementation are required to support it and should fail if they don't. The CTS should fail if they're missing and the CTS should not have to check for `undefined.

Reply:
got it! thanks for the explanation